rtrlib/rtr/packets.c

```diff
@@ -1310,10 +1465,49 @@ static int rtr_sync_receive_and_store_pdus(struct rtr_socket *rtr_socket)
 				}
 			}
 			RTR_DBG1("spki data added");
+			
+			// add all aspa pdu to the aspa_table
+			for (unsigned int i = 0; i < aspa_pdus_nindex; i++) {
```

Comment:
s/unsigned int/size_t/

Reply:
yup, just copied that from above, will change that